packages/web-components/fast-foundation/src/radio-group/radio-group.template.ts

```diff
@@ -11,6 +11,10 @@ export const RadioGroupTemplate = html<RadioGroup>`
         role="radiogroup"
         aria-disabled="${x => x.disabled}"
         aria-readonly="${x => x.readOnly}"
+        @keypress="${(x, c) => x.keypressHandler(c.event as KeyboardEvent)}"
+        @click="${(x, c) => x.clickHandler(c.event as MouseEvent)}"
+        @keydown="${(x, c) => x.keydownHandler(c.event as KeyboardEvent)}"
```

Comment:
We can't just use one of keydown/keypress?  Why both?

Reply:
well that seems like a valid question, looking into it

Reply:
Well it was never getting called anyway as far as I can tell, so just removed it, also fixed the tabbing bug that was caused by me moving event handlers to template. That gets me every time, wondering if that's the default we really want.